internal/internal_worker.go

```diff
@@ -574,6 +574,14 @@ func (th *hostEnvImpl) getRegisteredActivities() []activity {
 	return activities
 }
 
+func (th *hostEnvImpl) getRegisteredAcvitityNames() []string {
+	activityNames := make([]string, 0, len(th.activityFuncMap))
```

Comment:
As you are using slices, the more idiomatic way would be:

var activityNames []string
...
activityNames = append(activityNames, a.ActivityType().GetName())

return activityNames

or even:

func (th *hostEnvImpl) getRegisteredAcvitityNames() (activityNames []string) {
...
activityNames = append(activityNames, a.ActivityType().GetName())
...
}

Reply:
fixed